sockeye/arguments.py

```diff
@@ -1075,6 +1079,56 @@ def add_translate_cli_args(params):
     add_logging_args(params)
 
 
+def add_score_cli_args(params):
+    add_training_data_args(params, required=False)
+    add_vocab_args(params)
+    add_device_args(params)
```

Comment:
just out of curiosity: would scoring work with multiple gpus where each gpu gets a chunk of batch size?

Reply:
I haven't written any code to do this. Is it handled transparently by MXNet, or do you have to do it manually in train?

Reply:
Should just work then.